spec/requests/providers/capital_introductions_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Providers::CapitalIntroductionsController, type: :request do
+  let(:legal_aid_application) { create :legal_aid_application }
+  let(:provider) { legal_aid_application.provider }
+
+  describe 'GET /providers/applications/:id/capital_introduction' do
+    subject { get providers_legal_aid_application_capital_introduction_path(legal_aid_application) }
+
+    context 'when the provider is not authenticated' do
+      before { subject }
+      it_behaves_like 'a provider not authenticated'
+    end
+
+    context 'when the provider is authenticated' do
+      before do
+        login_as provider
+        subject
+      end
+
+      it 'returns http success' do
+        expect(response).to have_http_status(:ok)
+        expect(response.body).to include('Before you continue')
+      end
+
+      context 'when the Continue button is pressed' do
```

Comment:
This is a completely different spec - the fact that the subject has changed tells us that. So rather being a change of context within the GET action spec, this needs its own PATCH describe block outside of the GET describe block.